pkg/cri/nri/nri-api_other.go

```diff
@@ -0,0 +1,145 @@
+//go:build !linux
+
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package nri
+
+import (
+	"context"
+
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/containers"
+	cstore "github.com/containerd/containerd/pkg/cri/store/container"
+	sstore "github.com/containerd/containerd/pkg/cri/store/sandbox"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	cri "k8s.io/cri-api/pkg/apis/runtime/v1"
+
+	"github.com/containerd/containerd/pkg/cri/constants"
+	"github.com/containerd/containerd/pkg/nri"
+
+	"github.com/containerd/nri/pkg/api"
+)
+
+type API struct {
+}
+
+func NewAPI(nri.API) *API {
+	return nil
+}
+
+func (a *API) Register(CRIImplementation) error {
+	return nil
+}
+
+func (a *API) IsEnabled() bool {
+	return false
+}
+
+//
+// CRI-NRI lifecycle hook no-op interface
+//
+
+func (*API) RunPodSandbox(context.Context, *sstore.Sandbox) error {
+	return nil
```

Comment:
Does it have to be Linux only?
Sandboxed CRI expected to be cross-platform (as we isolate podsandbox containers).

Reply:
No, eventually not. But at the moment we don't have proper support for anything else than Linux in NRI itself.